Demo/src/main/java/com/paypal/android/api/model/CardPaymentToken.kt

```diff
@@ -1,8 +1,13 @@
 package com.paypal.android.api.model
 
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
 data class CardPaymentToken(
     val id: String,
     val customerId: String,
+    @SerialName("last_digits")
     val cardLast4: String,
```

Comment:
```suggestion
    val lastDigits: String,
```
I think it's worth renaming this property to `lastDigits` as well for consistency.